key.core/src/main/java/de/uka/ilkd/key/nparser/builder/ExpressionBuilder.java

```diff
@@ -1047,7 +1047,16 @@ public QuantifiableVariable visitOne_bound_variable(KeYParser.One_bound_variable
         if (sort != null && id != null) {
             return bindVar(id, sort);
         }
-        return doLookup(new Name(ctx.id.getText()), schemaVariables(), variables());
+
+        // TODO Does this not check schemaVariables() for the second time?
```

Comment:
Yes, I'd remove `schemaVariables()` below. I think the check above was added later.